### DISCUSSION TITLE
Adding a field for institutions which possess scans of manuscripts

### DISCUSSION THREAD

**Discussion by chakrabortydeepro** [PanditProject/Website#18] [Help]
Some institutions (such as the IGNCA, the National Mission for Manuscripts, the Centre for Studies in Social Sciences, the Endangered Archive Programme of the British Library) have scanned copies of manuscripts from other institutions. It would be great if we can add a field for such institutions. We can definitely add this information as a "discrete fact" but allowing it to be a separate field will help to know which institutions have scans of which manuscripts. 

Thanks,
Deepro

**Comment by chakrabortydeepro** [comment]
There is another way to do it with the present structure. We have an "alternative collection" field. We can name these collections of scans and can connect those collections to the respective institutions. Let me know if this is allowed because these collections won't be physical collections.

**Reply by AmirSimantov** [reply]
No, that field has another semantics.

**Comment by AmirSimantov** [comment]
Thanks, @chakrabortydeepro, for the idea.

Question:
Will adding an inner field "Available at (Institution)" to the "Resource" composite field in the **Manuscript** entity type address this need correctly, in your opinion?

**Reply by AmirSimantov** [reply]
Also, it would be beneficial to understand the incentive to add this data; is it because you find it in the sources you work on or/and you think it may help future researchers somehow?

**Reply by chakrabortydeepro** [reply]
I think we need to give an URL if we need to say something in the "Resource" field. But very often these digital copies are not available online. They are locally stored. In that case, I don't think adding an inner field "Available at (Institution)" inside the "Resource" filed would help.

**Comment by chakrabortydeepro** [comment]
Hello Amir,

Both actually. Many manuscripts which I work on are stored digitally in
certain other institutions. Other researchers should be benefited if they
know where they can find digital copies of the manuscripts they are looking
for.

Thanks,
Deepro

**Comment by AmirSimantov** [comment]
@ybronner is it in Pandit's scope? If yes, would you like it to be in the "Resource" field?

**Comment by wujastyk** [comment]
I don't want to complicate this issue unnecessarily.  Probably you can
ignore this comment.  But should we be considering scans as apographs?
Does that make a difference?  Professional librarians call scans (and
microfilms and photocopies) "surrogates."

**Reply by AmirSimantov** [reply]
A discussion is a correct place to raise ideas :)
Does your comment address only the naming, or does it have any semantic implication that might influence the structure?

**Comment by AmirSimantov** [comment]
@chakrabortydeepro 
How do you know about these scans? Where is it written? What other data you have about them? Location inside the institution? Format?

**Reply by chakrabortydeepro** [reply]
Dear @AmirSimantov 
Prof. @wujastyk gave the answer. I'm adding a couple of examples. 
1. surrogates not publicly available on the internet: http://ignca.gov.in/mss/JK_ORL_Srinagar_Persian.pdf (the IGNCA catalogue of digitized manuscripts from Oriental Research Library, Srinagar. 
2. surrogates publicly available on the internet:  https://eap.bl.uk/collection/EAP781-1-1/search (list of manuscripts from Bangiya Puran Parishad digitized by the Endangered Archive Programme of the British Library)

**Reply by AmirSimantov** [reply]
Thanks, @chakrabortydeepro.
I can see that @wujastyk opened a new thread below, so I have also read that thread.

@wujastyk - we talked about this topic about a year ago; you sent an email to @ybronner titled "microfilms numbers." There was a long discussion afterward, and in the end, we decided that a simple "Microfilm copy" text field in the Manuscript entity type will be enough. It followed a trial of planing an entity type that was withdrawn. I have shelved that effort which you can still see [as a tab](https://docs.google.com/spreadsheets/d/1SLjJyuPEU8BXfschex56HGRAA9M3AKjMWWlyq7cJf88/edit#gid=321422946) in the configuration spreadsheet. I understand that Deepro's need/idea raises it again. Apart from the decision whether to implement it now or not, we may reopen this topic. If yes, I will elaborate on that plan.

**Comment by wujastyk** [comment]
@AmirSimantov "How do you know about these scans?"  There are different ways.  For example, it is common knowledge amongst Indologists that the IGNCA in Delhi has photographs and scans of thousands of MSS from all over India.  Similarly, there are microfilms of the thousands of Nepalese MSS processed by the Nepal-German NGMPP project.  Similarly, the MSS at the University of Pennsylvania were microfiched many years ago. And so on.  This is information one picks up through visiting institutions, talking to colleagues, etc.

**Reply by wujastyk** [reply]
Following @chakrabortydeepro 's thought, there are two categories of surrogates that would be worth distinguishing in Pandit.

1. surrogates that are publicly available on the Internet.  PP should have a URL pointing to these surrogates.
2. surrogates that are known to exist (like IGNCA) but that are not on the Internet.  In such cases, it is useful to a PP reader to know that a surrogate exists, even if it can't be accessed directly online.

**Reply by AmirSimantov** [reply]
@wujastyk thanks, let's continue the thread above.
